File: M_Trivia/M_Trivia/CloseRoomResponse.cpp
#include "CloseRoomResponse.hpp"
#include "json.hpp"


CloseRoomResponse::CloseRoomResponse(CloseRoomResponseErrors* errors)
	: Response(errors)
{
}

ResponseCodes CloseRoomResponse::getCode() const
{
	return ResponseCodes::C_CloseRoomResponse;
}

File: ClientApp/ClientApp/Models/Responses/CloseRoomResponse.cs
using ClientApp.Models.ResponseErrors;
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ClientApp.Models.Responses
{
    class CloseRoomResponse : Response
    {
        private CloseRoomResponse() { }
        
        [JsonProperty]
        public CloseRoomResponseErrors Errors { get; private set; }
        public override ResponsesCodes GetCode() => ResponsesCodes.CloseRoomResponse;
    }
}
